cms/djangoapps/contentstore/signals/handlers.py

```diff
@@ -79,6 +146,27 @@ def listen_for_course_publish(sender, course_key, **kwargs):  # pylint: disable=
 
     update_discussions_settings_from_course_task.delay(course_key_str)
 
+    # Send to a signal for catalog info changes as well, but only once we know the transaction is committed.
+    transaction.on_commit(lambda: emit_catalog_info_changed_signal(course_key))
+
+
+@receiver(COURSE_CATALOG_INFO_CHANGED)
+def listen_for_course_catalog_info_changed(sender, signal, **kwargs):
+    """
+    Publish COURSE_CATALOG_INFO_CHANGED signals onto the event bus.
+    """
+    try:
+        # FIXME: This needs to be replaced with a configurable loading
+        # mechanism built into openedx-events.
+        # See https://github.com/openedx/openedx-events/pull/85)
+        from edx_event_bus_kafka.publishing.event_producer import send_to_event_bus
```

Comment:
Hmm, now that event bus kafka is a dependency, do we still want to use this pattern?

Reply:
Mmm, we can switch that to a top-import (though we'll just need to remove it again later once it's configurable).

Reply:
Should be fixed now.